scenario/mocking.py

```diff
@@ -134,12 +135,12 @@ def _get_secret(self, id=None, label=None):
             try:
                 return next(filter(lambda s: s.id == id, self._state.secrets))
             except StopIteration:
-                raise RuntimeError(f"not found: secret with id={id}.")
+                raise SecretNotFoundError(f"not found: secret with id={id}.")
         elif label:
             try:
                 return next(filter(lambda s: s.label == label, self._state.secrets))
             except StopIteration:
-                raise RuntimeError(f"not found: secret with label={label}.")
+                raise SecretNotFoundError(f"not found: secret with label={label}.")
```

Comment:
if the goal is to mimick what ops does, then there should be no message
```suggestion
                raise SecretNotFoundError()
```

Reply:
Done